src/frontend/src/pages/view-challenge/view-challenge.tsx

```diff
@@ -95,13 +85,14 @@ export const ViewChallenge: React.FC = () => {
                     </Grid>
                 </Grid>
                 <Grid item>
-                    <Typography variant="h4">Sentence:</Typography>
+                    <Typography variant="h6">Sentence</Typography>
                 </Grid>
                 <Grid item>
                     <Paper elevation={1}>
                         <Box p={3}>
                             <Typography>
-                                Word Scramble is a very felicific game!
+                                User submitted sentence that is really long and
+                                will fit to the screen
```

Comment:
Put this on one line. 
```suggestion
                                User submitted sentence that is really long and will fit to the screen
```